src/expandableCalendar/calendarProvider.js

```diff
@@ -13,15 +16,20 @@ class CalendarProvider extends Component {
     // Initial date in 'yyyy-MM-dd' format. Default = Date()
     date: PropTypes.any.isRequired,
     // callback for date change event
-    onDateChanged: PropTypes.func
+    onDateChanged: PropTypes.func,
+    // whether to show the today button
+    todayButton: PropTypes.bool
```

Comment:
maybe, name this prop `showTodayButton` so it's clear it's boolean.

Reply:
Done.